src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -74,7 +73,7 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
           pageCompiled,
           files,
           pageCompiled.allEventTeasers,
-          'event_listing.drupal.liquid',
+          'va-medical-centers/event_listing.drupal.liquid',
```

Comment:
Pages that are generated and need to have a specific layout applied need to also include the directory path, because they aren't using the `layouts/config` file.